app/scripts/modules/core/application/config/serializeApplicationSection.component.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.core.application.config.serialize.component', [
+    require('angular-ui-router'),
+    require('./../../serialize/serialize.write.service.js'),
+    require('../../confirmationModal/confirmationModal.service.js'),
+    require('../../overrideRegistry/override.registry.js'),
+  ])
+  .component('serializeApplicationSection', {
+    restrict: 'E',
```

Comment:
For the component config, you can drop `restrict`, `scope`, and `controllerAs` (by default, it will use `$ctrl` in the template).